src/Vgrep/Widget/Results.hs

```diff
@@ -69,8 +75,10 @@ pageDown = do
     modifying files (repeatedly moveDown)
 
 repeatedly :: (a -> Maybe a) -> a -> a
-repeatedly f a = case f a of Just a' -> repeatedly f a'
-                             Nothing -> a
```

Comment:
I don't think extracting a `go` adds to clarity. Besides, I liked the pattern guard version even better.